staging/src/github.com/openshift/oc/pkg/cli/admin/release/git.go

```diff
@@ -205,16 +198,21 @@ func mergeLogForRepo(g *git, repo string, from, to string) ([]MergeCommit, error
 		}
 
 		msg := records[3]
-		if m := reBug.FindStringSubmatch(msg); m != nil {
-			mergeCommit.Subject = msg[len(m[0]):]
-			mergeCommit.Bug, err = strconv.Atoi(m[1])
-			if err != nil {
+
+		// If it contains `[release-X.Y]` anywhere, just throw that away
```

Comment:
This doesn’t work for repos that have openshift-* branches (etcd, prometheus)

Reply:
Stripping the leading [xxx] segments until we hit something that isn’t guarded that way might be a compromise I could live with, although it’s a bit precious.

Reply:
It also wouldn’t work for hot-fix branches or those that were created for an embargo